File: Solutions by Programming Task/Fibonacci sequence/Java/Math.java
public class Main {
    private static int Fib(int x) {
        return (int) (Math.pow(2, -x)
                * (Math.pow(1 + Math.sqrt(5), x) - Math.pow(-1 + Math.sqrt(5), x) * Math.cos(Math.PI * x))
                / Math.sqrt(5));
    }

    public static void main(String[] args) {
        System.out.println(Fib(10));
    }
}

File: Solutions by Programming Task/Fibonacci sequence/Go/Math.go
package main

import (
	"fmt"
	"math"
)

var Fib = func(x int) int {
	return int(math.Round(
		math.Pow(2, float64(-x)) *
		(math.Pow(1 + math.Sqrt(5), float64(x)) -
			math.Pow(-1 + math.Sqrt(5), float64(x)) *
			math.Cos(math.Pi * float64(x))) /
		math.Sqrt(5)))
}

func main() {
	fmt.Println(Fib(10))
}
